### DISCUSSION TITLE
Results of one API response as header of another

### DISCUSSION THREAD

**Discussion by pipdata** [scalar/scalar#1201] [Q&A]
One API call in my scalar docs is an authorization request. It returns a JWT.
Is there a way to have the returned JWT (if not null) copied over the headers section of all the other requests? The point would be to avoid a lot of Ctrl-C/Ctrl-V.
Thanks.

**Comment by marclave** [comment]
@pipdata this is a fantastic idea!

I think we should most likely accomodate that once you set the auth, you can toggle a `global auth` field which will keep it across all requests.

needs some more thoughts but what do you think @hanspagel ?

**Reply by pipdata** [reply]
@marclave Thanks. Regarding the auth, is there a way to set the "Authorize" button like in (normal) swagger? I think that would work fine for what's discussed here. I can't find it in the docs and I apologize if I missed it.
